lib/Client.js

```diff
@@ -2071,6 +2076,36 @@ class Client extends EventEmitter {
         }));
     }
 
+    _formatAllowedMentions(allowed) {
+        if(allowed) {
```

Comment:
You could remove nesting by making this reverse
`if (!allowed) return this.options.allowedMentions`

Reply:
I don't see any benefit in doing this, this seems like a stylistic suggestion, and not one that matches the style eris uses.